src/catalog/catalog_entry/table_catalog_entry.cpp

```diff
@@ -91,6 +93,20 @@ void TableCatalogEntry::renameProperty(const std::string& propertyName,
     propertyCollection.rename(propertyName, newName);
 }
 
+std::string TableCatalogEntry::getDisplayLabelName(Catalog* catalog,
```

Comment:
I wonder if we should keep a `label` field in `TableCatalogEntry`, which is different from `name`.

`label` is defined by users, while `name` is used internally by us. Or maybe we even don't need `name` at all?